tsconfig.json

```diff
@@ -22,23 +22,24 @@
     "jsx": "react",
     "baseUrl": ".",
     "paths": {
-        "@mattermost/client": ["packages/client/src"],
+        "@mattermost/client/*": ["packages/client/src/*"],
         "@mattermost/components": ["packages/components/src"],
         "@mattermost/types/*": ["packages/types/src/*"],
         "mattermost-redux/*": ["packages/mattermost-redux/src/*"],
         "reselect": ["packages/reselect/src"],
-        "!!file-loader*": ["utils/empty-string"]
+        "!!file-loader*": ["utils/empty-string"],
+        "@e2e-support/*": ["e2e/playwright/support/*"],
+        "@e2e-test.config": ["e2e/playwright/test.config.ts"]
     }
   },
   "include": [
     "./**/*"
   ],
   "exclude": [
     "dist",
-    "e2e",
+    "e2e/cypress",
```

Comment:
Only enable for Playwright at `e2e/playwright` and remain `e2e/cypress` to be excluded until TS migration for Cypress is ready for type checking.